app/migrations/20240814014055-ActivityValue-backfill-activity-data-jsonb.cjs

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(`
+      UPDATE "ActivityValue"
+      SET "activity_data_jsonb" = "activity_data"::jsonb
+      WHERE "activity_data_jsonb" IS NULL; 
```

Comment:
I think you want IS NOT NULL here? Otherwise you just end up writing the null values to the new column.

Reply:
The Is NULL, ensures that we only fill in columns that are null. So we don't override any "activity_data_jsonb" field that has it's own values

Reply:
Ah fair, I misunderstood the purpose of the query. Makes sense then :thumbsup::skin-tone-3: